src/style-ast.ts

```diff
@@ -75,7 +75,7 @@ export class StyleAST {
 		const clone = new StyleAST( this.css, csstree.clone( this.ast ), this.errors );
 
 		clone.pruneMediaQueries();
-		clone.pruneKeyframes();
+		clone.pruneAtRules();
```

Comment:
going by naming only - wouldn't this remove `@media` too?

Reply:
No, it prunes unwanted At-rules, including `@keyframes`, `@charset` and `@import`. Would it help if it specifies unwanted in the fn name?

Reply:
I'd prefer `pruneAtRules(['keyframe', 'charset', 'import'])`. Barring that, "unwanted" might be ok too.

Reply:
![image](https://github.com/Automattic/jetpack-boost-critical-css-gen/assets/1369626/8385e08f-d065-473c-a8c4-f8362fed8158)

Yes, that's much cleaner. Applied in 289bcc6. Thanks :)